pkg/controller/yurtappdaemon/workloadcontroller/workload.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2021 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package workloadcontroller
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	unitv1alpha1 "github.com/openyurtio/openyurt/pkg/apis/apps/v1alpha1"
+)
+
+type Workload struct {
+	Name      string
+	Namespace string
+	Kind      string
+	Spec      WorkloadSpec
+	Status    WorkloadStatus
+}
+
+// WorkloadSpec stores the spec details of the workload
+type WorkloadSpec struct {
+	Ref          metav1.Object
+	Toleration   []corev1.Toleration
```

Comment:
Toleration --> Tolerations because a slice of `corev1.Toleration` are included.

Reply:
done